modules/web-console-query-cli.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * web_console/web-console-query-cli.adoc
+
+[id="web-console-query-cli_{context}"]
+= Query image vulnerabilities from the CLI
```

Comment:
Same about gerunds, so "Querying image vulnerabilities from the CLI"

Reply:
Yes.

Reply:
Done.